microprofile/service-common/src/main/java/io/helidon/microprofile/servicecommon/HelidonRestCdiExtension.java

```diff
@@ -254,6 +254,16 @@ protected HttpRouting.Builder routingBuilder(ServerCdiExtension server) {
                 : server.serverNamedRoutingBuilder(routingName);
     }
 
+    /**
+     * Returns the config key for settings for the specified suffix nested within the server config tree.
```

Comment:
Minor comment: might be nice to append something like: "which is `server.features.observe.observers.`."